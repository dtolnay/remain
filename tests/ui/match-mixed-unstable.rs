#![feature(proc_macro_hygiene, stmt_expr_attributes)]

use remain::sorted;

fn main() {
    let value = "trivial_regex";

    #[sorted]
    match value {
        "cognitive_complexity" => {}
        "hello world" => {}
        "implicit_hasher" => {}
        "inefficient_to_string" => {}
        "integer_division" => {}
        "large_digit_groups" => {}
        let_it_be if false => {}
        "let_unit_value" => {}
        "manual_map" => {}
        "match_bool" => {}
        mixed_in if false => {}
        "needless_pass_by_value" => {}
        "new_ret_no_self" => {}
        "nonstandard_macro_braces" => {}
        "option_if_let_else" => {}
        "option_option" => {}
        "rc_buffer" => {}
        "string_lit_as_bytes" => {}
        "trivial_regex" => {}
        "useless_let_if_seq" => {}
        "trivially_copy_pass_by_ref" => {}
        "unnested_or_patterns" => {}
        "unreadable_literal" => {}
        "unsafe_vector_initialization" => {}
        _ => {}
    }
}
