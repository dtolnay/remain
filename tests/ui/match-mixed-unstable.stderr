error: trivially_copy_pass_by_ref should sort before useless_let_if_seq
 --> $DIR/match-mixed-unstable.rs:8:5
  |
8 |     #[sorted]
  |     ^^^^^^^^^
  |
  = note: this error originates in the attribute macro `sorted` (in Nightly builds, run with -Z macro-backtrace for more info)
